DatadogInternal/Sources/Models/SessionReplay/SessionReplayConfiguration.swift

```diff
@@ -32,9 +32,10 @@ public enum SessionReplayPrivacyLevel: String {
 ///     )
 ///
 public protocol SessionReplayConfiguration {
-    // TODO: RUM-5766 - Pass correct privacy level to webviews
-    /// Deprecated
-    /// Former global privacy level to use in Session Replay.
+    /// Global privacy level to use in Session Replay.
+    /// Although this setting is deprecated on mobile,
+    /// it is still needed to configure the browser SDK,
+    /// which currently does not support fine-grained privacy options.
     var privacyLevel: SessionReplayPrivacyLevel { get }
 }
```

Comment:
/suggestion We have now privacy levels defined in 2 places, the configuration flow is not super clear and it won't be easy to change if Browser SDK ever support fine-grained privacy. IMO, we should follow the pattern in place:
1. Define all privacy levels in `DatadogInternal/Sources/Models/SessionReplay/SessionReplayConfiguration.swift`
2. Move `determineWebViewPrivacyLevel` to `WebViewTracking` domain and make the conversion there

This way, if we have to inject fine-grained privacy options to the JS bridge, we would only have to change `WebViewTracking` module.

Reply:
It makes sense. But I need `determineWebViewPrivacyLevel` in `SessionReplay.Configuration` to assign a value to `defaultPrivacyLevel` when using the new API. Effectively, it's not being used except for the webview integration, so I could make `defaultPrivacyLevel` `optional` so that it doesn't require a value. What do you think?

Reply:
Or add a default value to `defaultPrivacyLevel`, which is kind of what we already do in the API.
`public var defaultPrivacyLevel: SessionReplayPrivacyLevel = .mask`